<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>database</class>
 <widget class="QWidget" name="database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>60</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/data_nie.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="now_bt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>547</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(132, 66, 198);
border-radius:10px;		 
background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
   </property>
   <property name="text">
    <string>NOW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayAll_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(180, 150, 0);   
border-radius:10px;		 
background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);			
font-size:15px; 		 
</string>
   </property>
   <property name="text">
    <string>显示所有</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_l">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>550</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(140, 66, 0);
border-radius:10px;		 
background-color: rgb(198, 66, 0);
color: rgb(255, 255, 255);
 


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>517</y>
     <width>194</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>7</day>
    </date>
   </property>
   <property name="time">
    <time>
     <hour>20</hour>
     <minute>50</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>509</y>
     <width>3</width>
     <height>86</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>550</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(140, 66, 0);
border-radius:10px;		 
background-color: rgb(198, 66, 0);
color: rgb(255, 255, 255);
 


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>497</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(151, 151, 227);
border-radius:10px;		 
background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);
font-size:16px; 		 

</string>
   </property>
   <property name="title">
    <string>日期查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="currDay_bt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(0, 120, 180);
border-radius:10px;		 
background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>最近一天</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currThDay_bt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(63, 63, 190);
border-radius:10px;		 
background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>最近三天</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currWeek_bt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(180, 0, 180);
border-radius:10px;		 
background-color: rgb(255, 0, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 




</string>
    </property>
    <property name="text">
     <string>最近一周</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clear_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>578</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(134, 134, 0);   
border-radius:10px;		 
background-color: rgb(170, 170, 0);
color: rgb(255, 255, 255);			
</string>
   </property>
   <property name="text">
    <string>清空数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_val_bt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>580</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(0, 133, 120);
border-radius:10px;		 
background-color: rgb(0, 170, 255);	
color: rgb(255, 255, 255);
font-size:15px; 		 

</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(100, 100, 194);
border-radius:10px;		 
background-color: rgb(129, 129, 194);
color: rgb(255, 255, 255);



</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_data_bt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>517</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(0, 133, 120);
border-radius:10px;		 
background-color: rgb(0, 170, 255);	
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_l">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>519</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>547</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>7</day>
    </date>
   </property>
   <property name="time">
    <time>
     <hour>20</hour>
     <minute>50</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>数据库管理平台</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>505</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(59, 59, 208);   
border-radius:10px;		 
background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);			

</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>3</width>
     <height>86</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>494</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(80, 229, 229);
border-radius:10px;		 
background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);
font-size:15px; 		 

</string>
   </property>
   <property name="title">
    <string>值查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>tableView</zorder>
  <zorder>icon</zorder>
  <zorder>now_bt</zorder>
  <zorder>displayAll_bt</zorder>
  <zorder>end_l</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>dateTimeEdit</zorder>
  <zorder>line_2</zorder>
  <zorder>spinBox</zorder>
  <zorder>clear_bt</zorder>
  <zorder>select_val_bt</zorder>
  <zorder>comboBox</zorder>
  <zorder>select_data_bt</zorder>
  <zorder>start_l</zorder>
  <zorder>dateTimeEdit_2</zorder>
  <zorder>title</zorder>
  <zorder>exit_bt</zorder>
  <zorder>line</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
